src/components/NFTBook/TippingDialog.vue

```diff
@@ -50,16 +58,54 @@
               <Label class="text-dark-gray" preset="h4">{{ tipping }}</Label>
               <Label preset="h6" class="text-medium-gray">{{ currency }}</Label>
             </div>
-          </ButtonV2>
+            <div
+              v-if="selectedIndex === i && !inputValue"
+              :class="[
+                'w-[20px]',
+                'h-[20px]',
+                'flex',
+                'justify-center',
+                'items-center',
+                'rounded-[50%]',
+                'bg-like-green',
+                'border-[1px]',
+                'border-white',
+                'absolute',
+                'top-[-10px]',
+                'left-[-10px]',
+              ]"
+            >
+              <IconCheckMini class="text-white" />
```

Comment:
How about making the SVG absolute and removing the parent div

Reply:
The parent div contains a circular background